content/en/docs/setup/best-practices/certificates.md

```diff
@@ -1,8 +1,9 @@
 ---
```

Comment:
Some of what's in this page is about good practice, and some is simply explaining why you need certificates at all (even if you go with kubeadm's default settings).

It seems to me like both parts are worthwhile:
- concept-style documentation covering certificates and how a Kubernetes cluster uses them
  - eg, readers deploying in VMs on their laptop would find this useful.
- recommendations on setting up CAs & issuing certificates for real world deployments, so that they are suitably secure and amenable to automation

Can we have the best of both worlds?

Reply:
Yes, we need the best of both worlds and may be refactor the content to update Concepts and Getting started tabs. How about a followup PR if we agree on the outline under Getting started?